### PR TITLE
[WAZO-793] Allow to completly remove the docker env

### PR DESCRIPTION
**Pull request by sileht**
Pass $asset argument to exit/restart/reset
Allow to completly remove the docker env

### CODE REVIEW

File: denv [hunk 1]
```diff
@@ -7,13 +7,16 @@ denv () {
...
         ;;
         r|restart)
-            denv_restart $2
+            denv_restart $2 $3
         ;;
         s|reset)
-            denv_reset $2
+            denv_reset $2 $3
+        ;;
+        r|remove)
```
Comment (sduthil) [inline denv]: There's a conflict on shortcut "r"

Comment (sduthil) [reply denv]: Also, what's the difference with denv_exit?

Comment (sileht) [reply denv]: I changed `remove` to `delete`.

`denv_exit` just stop the container (you sill see them as stopped with "docker ps -a")
`denv_delete` will also delete them and delete the associated volumes

Comment (sduthil) [reply denv]: I don't see the different use case for `delete` and `exit`. How about merging them?

Comment (sileht) [reply denv]: That's work for me.

**Pull request merged**